multiraft/multiraft.go

```diff
@@ -586,7 +586,7 @@ func (s *state) start() {
 							log.Infof("node %v: got message for unknown group %d; creating it", s.nodeID, req.GroupID)
 						}
 						if err := s.createGroup(req.GroupID, req.ToReplica.ReplicaID); err != nil {
-							log.Warningf("Error creating group %d: %s", req.GroupID, err)
+							log.Warningf("Error creating group %d in response to incoming message: %s", req.GroupID, err)
```

Comment:
a bit misleading since the thing that follows the colon is the error not the incoming message.